src/Jupyter/ConfigurationSource.cs

```diff
@@ -96,6 +96,13 @@ public interface IConfigurationSource
         /// </summary>
         public int TraceVisualizationDefaultDepth =>
             GetOptionOrDefault("trace.defaultDepth", 1);
+
+        /// <summary>
+        ///     Allows for setting the default visualization style for visualizing Q# operations
+        ///     using the <c>%trace</c> command.
+        /// </summary>
+        public string TraceVisualizationStyle =>
+            GetOptionOrDefault("trace.style", "Default");
```

Comment:
Could you make this an enum, similar to `BasisStateLabelingConvention` and `PhaseDisplayStyle` above? A bonus is that this gives you some input validation for free.